docs/mixin/dashboards/promscale.json

```diff
@@ -2513,7 +2513,7 @@
               },
               "editorMode": "code",
               "exemplar": false,
-              "expr": "max(promscale_sql_database_worker_maintenance_job_long_running_total{namespace=~\"$namespace\"})",
+              "expr": "max(promscale_sql_database_worker_long_running_maintenance_jobs{namespace=~\"$namespace\"})",
```

Comment:
Why change from `promscale_sql_database_worker_maintenance_job_long_running_total` to
`promscale_sql_database_worker_long_running_maintenance_jobs` and not to `promscale_sql_database_worker_maintenance_job_long_running`?

IMHO it is better to keep a consistent prefix of `promscale_sql_database_worker_maintenance_` to allow easier query (think `{__name__=~"promscale_sql_database_worker_maintenance_.*"}` as well as easier code navigation as `promscale_sql_database_worker_maintenance_job` could be defined in the same place regardless of how many metric registries are there.
That said I am not tied to having it either way.